conf/webknossos.versioned.routes

```diff
@@ -4,6 +4,7 @@
 # Note: keep this in sync with the reported version numbers in the utils.ApiVersioning trait
 
 # version log:
+ # changed in v9: Datasets are now identified by their id, not their name.
```

Comment:
please add a slightly more verbose explanation for the concrete api changes here